src/idpi/operators/support_operators.py

```diff
@@ -122,4 +122,5 @@ def get_grid_coords(n: int, x0: float, dx: float, dim: str) -> xr.DataArray:
         A 1-D field containing the coordinates of the grid along the given dimension
 
     """
-    return xr.DataArray(np.arange(n, dtype=np.float32) * dx + x0, dims=dim)
+    values = np.arange(n) * dx + x0
+    return xr.DataArray(np.round(values, 6), dims=dim)
```

Comment:
Perhaps it would make sense to add a comment on why you round to 6 decimals in the description of the function.